utilities/BuildUtilities.groovy

```diff
@@ -648,6 +665,24 @@ def assertDbbBuildToolkitVersion(String currentVersion){
 	}
 }
 
+def retrieveHFSEncoding(File file) {
```

Comment:
Please add a description to the method. 

Additionally, I would prefer the method name as `retrieveHFSFileEncoding`

Reply:
I have added a method description and changed the name as requested.